pkg/acsengine/engine_test.go

```diff
@@ -371,3 +371,184 @@ func TestGetGPUDriversInstallScript(t *testing.T) {
 		t.Fatalf("VMSize without GPU should not receive a script, expected empty string, received %v", s)
 	}
 }
+func TestIsNSeriesSKU(t *testing.T) {
+	// VMSize with GPU and NVIDIA agreement for drivers distribution
```

Comment:
nit: should simply read "VMSize with GPU" instead. We only have agreement for NC6, NC12, NC24, NV6, NV12 and NV24 so far.  This is very minor, but I don't want anyone to get confused on this agreement stuff and distributes drivers without NVIDIA approval by mistake.